packages/scene-composer/src/components/SceneComposerInternal.tsx

```diff
@@ -39,25 +41,26 @@ export const SceneComposerInternal: React.FC<SceneComposerInternalProps> = ({
   // label body as being used by the scene. this allows other components to identify it vs. other page components
   document.body.className = document.body.className + ' ' + SCENE_BODY_CLASS;
 
-  useEffect(() => {
-    if (config.colorTheme === 'light') {
-      applyMode(Mode.Light);
-    } else {
-      applyMode(Mode.Dark);
-    }
-  }, [config.colorTheme]);
+  useAwsLightDarkModes(sceneComposerInternalRef, config.colorTheme === 'light' ? Mode.Light : Mode.Dark);
```

Comment:
NIT: 'light' can be replaced with Mode.Light
There is usage of hard-coded 'light' at line#39 as well.

Reply:
I would except that's an external API change where config.colorTheme is using colorTheme?: 'dark' | 'light'; not colorTheme?: Mode.  I'd like us to look at changing that when we upgrade the composer to full light mode support?

Reply:
I was suggesting that in this validation `config.colorTheme === 'light' ` instead of using a hard-coded text 'light', we could use a constant... maybe Mode.Light as its value is 'light'.
Hard-coded text 'light' is used at line #39 as well.

Reply:
Got your point.. It is ok for now 👍